plugins/modules/efs.py

```diff
@@ -474,6 +498,27 @@ def update_file_system(self, name, throughput_mode, provisioned_throughput_in_mi
                     self.module.fail_json_aws(e, msg="Unable to update file system.")
         return changed
 
+    def update_lifecycle_policy(self, name, transition_to_ia):
+        """
+        Update filesystem with new lifecycle policy.
+        """
+        changed = False
+        state = self.get_file_system_state(name)
+        if state in [self.STATE_AVAILABLE, self.STATE_CREATING]:
+            fs_id = self.get_file_system_id(name)
+            current_policies = self.connection.describe_lifecycle_configuration(FileSystemId=fs_id)
+            if transition_to_ia == 'None':
+                LifecyclePolicies = []
+            else:
+                LifecyclePolicies = [{ 'TransitionToIA': 'AFTER_' + transition_to_ia + '_DAYS' }]
```

Comment:
```suggestion
                LifecyclePolicies = [{'TransitionToIA': 'AFTER_' + transition_to_ia + '_DAYS'}]
```

To ensure consistent formatting of code in the collection we generally follow PEP8 for formatting: https://www.python.org/dev/peps/pep-0008/#whitespace-in-expressions-and-statements